web/lib/admin/MapNone.php

```diff
@@ -48,10 +48,10 @@ public function htmlHeadCode() {
         // no magic required if you want to nothing at all.
         return "<script>
             function locateMe() {
-                navigator.geolocation.getCurrentPosition(locate_succes,locate_fail,{maximumAge:3600000, timeout:5000});
+                navigator.geolocation.getCurrentPosition(locate_success,locate_fail,{maximumAge:3600000, timeout:5000});
             }
 
-            function locate_succes(p) {
+            function locate_success(p) {
```

Comment:
Is this function part of an API that cannot be changed without breaking compatibility?